### PR TITLE
DAM-174 Initial PWM terraform

### PR DESCRIPTION
**Pull request by marcus-bcl**

### CODE REVIEW

File: pwm/user_data/user_data.sh [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+- hosts: localhost
+  vars_files:
+   - "{{ playbook_dir }}/users.yml"
+   - "{{ playbook_dir }}/vars.yml"
+  roles:
+     - bootstrap
+     - users
+     - pwm
+EOF
+cat << EOF > /etc/sudoers.d/webops
```
Comment (smjmoj) [inline pwm/user_data/user_data.sh]: This is redundant, the users role takes care of this.

File: pwm/user_data/user_data.sh [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+# Install tools
+sudo -i
+yum install -y git wget yum-utils jq
+easy_install pip
+PATH=/usr/local/bin:$PATH
+pip install ansible==2.6 virtualenv awscli boto botocore boto3 passlib
+
+# Download user inventory and run ansible
+/usr/bin/curl -o ~/users.yml https://raw.githubusercontent.com/ministryofjustice/hmpps-delius-ansible/master/group_vars/dev.yml
+sed -i '/users_deleted:/,$d' ~/users.yml
```
Comment (smjmoj) [inline pwm/user_data/user_data.sh]: Should this be removed? Do we want to leave the default users (ec2-user or centos) on?

File: pwm/user_data/user_data.sh [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+# Install tools
+sudo -i
+yum install -y git wget yum-utils jq
+easy_install pip
+PATH=/usr/local/bin:$PATH
+pip install ansible==2.6 virtualenv awscli boto botocore boto3 passlib
+
+# Download user inventory and run ansible
+/usr/bin/curl -o ~/users.yml https://raw.githubusercontent.com/ministryofjustice/hmpps-delius-ansible/master/group_vars/dev.yml
```
Comment (smjmoj) [inline pwm/user_data/user_data.sh]: needs a parameter
group_vars/${bastion_inventory}.yml

File: pwm/user_data/user_data.sh (hunk 1 above)
Comment (marcus-bcl) [reply pwm/user_data/user_data.sh]: Removed

File: pwm/user_data/user_data.sh (hunk 2 above)
Comment (marcus-bcl) [reply pwm/user_data/user_data.sh]: According to Brett, the users_deleted part of the users role breaks on the ECS AMI - so recommendation was to remove it from the users.yml file

File: pwm/user_data/user_data.sh (hunk 3 above)
Comment (marcus-bcl) [reply pwm/user_data/user_data.sh]: Added bastion_inventory var

File: pwm/user_data/user_data.sh (hunk 2 above)
Comment (smjmoj) [reply pwm/user_data/user_data.sh]: a that would make sense. A comment to explain this would be useful for those coming along later :)

**Pull request merged**